common/source/docs/common-paramosd.rst

```diff
@@ -0,0 +1,134 @@
+
+Accessing parameter screens
+===========================
+
+To enable the parameter screens, set :ref:`OSD5_ENABLE <OSD5_ENABLE>` and :ref:`OSD6_ENABLE <OSD6_ENABLE>` = 1.
+Paramater screens can be accessed in the same way the conventional osd screens are accessed. 
+See :ref:`screens-and-screen-switching <common-osd-overview.html#screens-and-screen-switching>` for reference.
+When one of the parameter screens is selected, adjustments can be made in disarmed state only. If a parameter screen is 
+selected inflight, the screen allows display only. This allows live monitoring of automatic parameter adjustments, 
+i.e. when performing autotune procedures or automatic servo trims.
+
+
+Menu navigation
+===============
+
+Use the RC transmitter's roll and pitch input to navigate through the parameter screen. UP / DOWN and LEFT / RIGHT will navigate through 
+the screen's parameter slots. The selected parameter will blink. Move right on the slected parameter to adjust its value using 
+up or down stick input. Leave the selection with left roll stick input. On a typical mode 2 type transmitter this would be the right 
+gimbal stick:
+
+.. image:: ../../../images/stickinput.jpg
+    :target: ../_images/stickinput.jpg
+    :width: 450px
+
+  
+Parameter slot scroll-selection
+===============================
+
+Holding roll right for > 3 seconds on a parameter's value will highlight both the selected parameter and its value (blink). This allows 
+to scroll through the full list of parameters and place another paramater into that respective slot using pitch up and down stick input. 
+Move left to exit parameter selection and got back to standard screen navigation.
+
+.. note::
+
+   OSD based parameter adjustment is designed to allow basic adjustments without any aditional data link or GCS setup required. However, 
+   some paramaters with wide value ranges might not be ideal for adjustment using parameter OSD.
+
+
+Parameter value and range mnemonics
+===================================
+
+OSD based parameter screens use mnemonics to account for complex functional parameter values and value ranges. This allows estimate useful 
+incrementation steps for individual paramater value ranges, as well as to display human-readable abbreviations for: 
+
+- flightmodes
+
+- servo functions
+
+- serial protocols
+
+- RC aux options
+
+- failsave actions
+
+.. note::
+
+   This feature my have limitations on certain parameters with exceptionally wide value ranges and adaptive incrementation steps. Selecting a 
+   different parameter using scroll selection will reset the type mnemonics to "none".
+
+
+Populating a screen's parameter slots using maxproxy's OSD module
+=================================================================
+ 
+OSD based parameter setting includes a `mavproxy module <https://ardupilot.org/mavproxy/>`__ to populate each screen's parameter slots.
+Connect your flight controller and load the OSD module by typing :code:`module load osd`
```

Comment:
Connect your flight controller, connect to MAVProxy, and....

Reply:
fixed